src/api/todos.ts

```diff
@@ -1,8 +1,68 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
-  return client.get<Todo[]>(`/todos?userId=${userId}`);
+const showErrorNotification = (errorMessage: string) => {
+  const notification = document.querySelector('.notification');
```

Comment:
no more querySelector in react.js plz)) create/modify/control components using the react, do not work with dom by yourself